src/main/scala/sbtecr/EcrPlugin.scala

```diff
@@ -7,6 +7,8 @@ import sbt._
 import scala.language.postfixOps
 
 object EcrPlugin extends AutoPlugin {
+  
+  lazy val version:SettingKey[String] = settingKey[String]("The ECR docker image version.")
```

Comment:
Can we place this below `localDockerImage` in the `autoImport` object for consistency?